vcbuild.bat

```diff
@@ -51,6 +51,7 @@ set "common_test_suites=%js_test_suites% doctool addons addons-napi&set build_ad
 set http2_debug=
 set nghttp2_debug=
 set link_module=
+set exe_only=
```

Comment:
That's inconsistent with other arguments which looks like `<feature>` or `no<feature>`. How about `nocctest` instead? Or even better - maybe build `cctest` only if the `test` argument is specified?